README.rst

```diff
@@ -0,0 +1,50 @@
+Google Auth Python Library (Experimental)
+=========================================
+
+.. image:: https://travis-ci.org/GoogleCloudPlatform/google-auth-library-python.png?branch=master
+        :alt: Build status on Travis
+        :target: https://travis-ci.org/GoogleCloudPlatform/google-auth-library-python
+
+.. image:: https://readthedocs.org/projects/google-auth/badge/?version=latest
+        :alt: Documentation Status
+        :target: https://google-auth.readthedocs.io/en/latest/
```

Comment:
See the fancy RST syntax we use in `google-cloud-python`. Puts this content out of sight